deploy/helm/smg/templates/grafana-dashboard.yaml

```
@@ -0,0 +1,56 @@
+{{- if .Values.grafana.dashboard.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "smg.fullname" . }}-grafana-dashboard
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)